src/edu/washington/escience/myria/operator/network/partition/HowPartitioned.java

```diff
@@ -0,0 +1,60 @@
+package edu.washington.escience.myria.operator.network.partition;
+
+import java.util.Set;
+
+import javax.annotation.Nullable;
+
+import jersey.repackaged.com.google.common.collect.ImmutableSet;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.primitives.Ints;
+
+/**
+ * How a dataset is partitioned. There are two fields being recorded for now, the partition function and the sequence of
+ * worker IDs. More information can be added in the future if needed.
+ */
+public class HowPartitioned {
```

Comment:
This name seems really odd to me. Why not just something like PartitionInfo or PartitioningInfo?